ci-operator/step-registry/ovn/conf/ipsec-manifest/ovn-conf-ipsec-manifest-commands.sh

```diff
@@ -3,6 +3,35 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+cp "${CLUSTER_PROFILE_DIR}/pull-secret" /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret "${RELEASE_IMAGE_LATEST}" --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
+ocp_major_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $1}' )
+ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+
+if (( ocp_minor_version >= 15 && ocp_major_version == 4 )); then
```

Comment:
glad you figured this out @pperiyasamy.

I think it would be a little cleaner if you first made this ${SHARED_DIR}/manifest_cluster-network-03-config.yml
file with the < 4.14 contents and then just tweaked the ipsecConfig.mode value inline. will make it a little easier
to understand what the changes are. total nit though and LMK if you don't want to worry about it.


something like this should work:
`yq e '.spec.defaultNetwork.ovnKubernetesConfig.ipsecConfig.mode = "Full"' -i ${SHARED_DIR}/manifest_cluster-network-03-config.yml`